src/main.ts

```diff
@@ -15,6 +18,8 @@ export const appBindings = new ContainerModule((bind: interfaces.Bind) => {
     bind<ILoggerService>(DI_TYPES.LoggerService).to(LoggerService).inSingletonScope();
     bind<IUserService>(DI_TYPES.UserService).to(UserService).inSingletonScope();
     bind<UserController>(DI_TYPES.UserController).to(UserController).inSingletonScope();
+    bind<IEmployeeService>(DI_TYPES.EmployeeService).to(EmployeeService).inSingletonScope();
+    bind<EmployeeController>(DI_TYPES.EmployeeController).to(EmployeeController).inSingletonScope();
```

Comment:
shouldn't here be a IEmployeeController interface? like with services? I just repeated this line after your UserController